widget/playground/src/containers/StyleLayout/StyleLayout.General.tsx

```diff
@@ -139,6 +142,7 @@ export function General() {
           </FieldTitle>
           <Divider size={16} />
           <Slider
+            color={colors?.[mode]?.secondary || '#2284ED'}
```

Comment:
I think Slider component should have a default color instead of passing a hex code color if 'secondary' is undefined.